core/domain/html_cleaner_test.py

```diff
@@ -158,7 +158,9 @@ def test_get_rte_components(self):
         test_data = (
             '<p>Test text&nbsp;'
             '<oppia-noninteractive-math '
-            'raw_latex-with-value="&amp;quot;\\frac{x}{y}&amp;quot;">'
+            'math_content-with-value="{&amp;quot;raw_latex&amp;quot;:'
+            '&amp;quot;\\frac{x}{y}&amp;quot;,&amp;quot;svg_filename&amp;quot;:'
+            '&amp;quot;&amp;quot;}">'
```

Comment:
Should we add valid data for svg_filname in the test? [OR are you planning to change this once we have svg_filenames for all the math expressions?)

Reply:
I think It's better to add them once we finalize the naming conventions for math SVGs

Reply:
So once we will have the SVG for all the expressions are we going to have a strict validation for svg_filenmae to not be '' (empty string)?

Reply:
Yeah there will be an audit job.
Although the yaml files need to be changes manually